client/src/app/components/manage-organizations/organization-list/manage-organizations.component.ts

```diff
@@ -89,6 +89,16 @@ export class ManageOrganizationsComponent implements OnInit {
     this.isActive = organization.status ? 'active' : 'disabled';
   }
 
+  isGlobal(address: string): string {
+    const bracketCount = address.split('[').length - 1;
+    if (bracketCount > 3) {
+      return 'GLOBAL';
+    } else {
+      // Remove open and close bracket and apostrophe from the address string.
+      return address.replace(/\[|\]|\'/g, '');
+    }
+  }
+
```

Comment:
Are we sure brackets will always be the location delimiter ?

Reply:
100% Because we are storing it as a nested list on the backend but retrieving it as string (removing the brackets, apostrophe) for the the frontend.